### PR TITLE
Code Buddy - API documentation

### LINKED ISSUE #7: [3pt] - Add API documentation
Add full documentation for your API

### LINKED ISSUE #12: [1pt] - Deploy the app
- Deploy the app so you can use the actual API endpoints for requests

### PR DESCRIPTION
**Pull request by Donard97**
**I opened this PR for you to review my code if it is as requested. 👁️** 

### Project requirements that are done:

- Add API Documentation ✔️ 
image

- Add data for meeting type ✔️
- Update README file with all details in it ✔️
- Deploy the app ✔️

<hr>

Feel free to review my project and if you have any questions please ask me! 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,95 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Donard97/code-buddy-backend/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Hats off to [Murat Korkmaz](https://www.behance.net/muratk) for designing the awesome template that was used. ✨
```
Comment (ahzia) [inline README.md]: This is for Frontend Repo

File: README.md [hunk 2]
```diff
@@ -1 +1,95 @@
...
+> This project is about creating API end point using Ruby on Rails and deploying the API on heroku. The API has meetings and reservations endpoints. The API is used in our front end React project which is based on an app to book a meeting to learn or for coding session with your buddy. 
+
+
+## The learning objective of this project are
+
+- Create rails app
+- Connect react front end app to rails back end
+- Use API end point to get data
+- Fetch data from the API
+
```
Comment (ahzia) [inline README.md]: Add the Er Diagram, and mention which features are implemented and which features will be implemented in feature

File: README.md [hunk 3]
```diff
@@ -1 +1,95 @@
...
+## Kanban board
+
+- Click this [link](https://github.com/Donard97/code-buddy-backend/projects/1) to view the Kanban board that was used while building this app.
+
+- Click [here](https://user-images.githubusercontent.com/74506933/150403327-e2696816-3a09-41f5-b49f-a012d8cec8bb.png) to view the initial state of our Kanban board.
+
+- At the end of this project, there are two(2) people in the team.
+
+
```
Comment (ahzia) [inline README.md]: Add Frontend repo link

**Review by ahzia — CHANGES_REQUESTED** [review]
Hi @Donard97,

Good job so far!
There are some issues that you still need to work on!

not so close

## Required Changes ♻️

- Documentation is not deployed properly [it seems this may fix after merge]
- You need to update some details in the Readme file (Check inline comments)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md (hunk 1 above)
Comment (Donard97) [reply README.md]: Thank you for your feedback! I did it 💪

File: README.md (hunk 3 above)
Comment (Donard97) [reply README.md]: Thank you for letting me know. I add it.

File: README.md (hunk 2 above)
Comment (Donard97) [reply README.md]: Thank you for your feedback. I did it

**Comment by ahzia** [comment]
# Hi @Donard97  , 👏 👏 

Well done! 💯 💯 💯 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
giphy (10)

Cheers and Happy coding!👏👏👏

## Optional suggestions

- Add validations for models

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_**Feel free to leave any questions or comments in the PR thread if something is not 100% clear.**_

**Pull request merged**